reflections_ai/settings.py

```diff
@@ -69,6 +73,12 @@
 
 WSGI_APPLICATION = 'reflections_ai.wsgi.application'
 
+CORS_ORIGIN_WHITELIST = (
+    'google.com',
+    'hostname.example.com',
+    'localhost:4200',
+    '127.0.0.1:9000'
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this IP "127.0.0.1" address configurable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=python%3AS1313)